lib/core/src/lib/form/components/widgets/core/form-field.model.ts

```diff
@@ -414,7 +427,9 @@ export class FormFieldModel extends FormWidgetModel {
             }
             case FormFieldTypes.RADIO_BUTTONS: {
                 const radioButton: FormFieldOption = this.options.find((opt) => opt.id === this.value);
-                this.form.values[this.id] = radioButton || null;
+
+                this.form.values[this.id] = radioButton ? { ...radioButton, options: this.options } : { id: null, name: null, options: this.options };
+
                 break;
```

Comment:
The most important part, related to this comment:
![image](https://github.com/user-attachments/assets/739e06b0-95a7-4cae-9fa4-50c886d493db)

Reply:
We used **options** word when defining object representing radio buttons value but we can change it if needed (can be slightly confusing I guess) e.g. allOptions, optionContext, optionsOnSubmit

Reply:
**Update:** 
This convention will be followed only by radio buttons of **rest** type, for **manual** ones we will stick with the old implementation. 

Reason: 
- Manual radio can't store **options** property (runtime issue when submitting form) (no problems in case of **rest** type)
- Actually storing options for **manual** type doesn't have any sense, when specific process is triggered the static options list is fixed (releasing new version of the project can't impact this list e.g. when we will change options)
- Static options don't have to be requested in separate call, we have them out of the box